modules/claims_api/spec/requests/v2/veterans/power_of_attorney/2122a_spec.rb

```diff
@@ -152,19 +171,32 @@
                 end
 
                 context 'and the request includes a claimant' do
-                  it 'enqueues the PoaAssignDependentClaimantJob and not the PoaFormBuilderJob' do
+                  it 'enqueues the PoaFormBuilderJob' do
                     VCR.use_cassette('claims_api/mpi/find_candidate/valid_icn_full') do
                       mock_ccg(scopes) do |auth_header|
-                        expect do
-                          post appoint_individual_path, params: claimant_data.to_json, headers: auth_header
-                        end.to change(ClaimsApi::PoaAssignDependentClaimantJob.jobs, :size).by(1)
-
                         expect do
                           post appoint_individual_path, params: claimant_data.to_json, headers: auth_header
                         end.not_to change(ClaimsApi::V2::PoaFormBuilderJob.jobs, :size)
```

Comment:
How is this test passing now? Shouldn't the `PoaFormBuilderJob` queue increase by one?

Reply:
Another good catch.  Appears there is a failure in the test run prior to that line in the controller, so the job is not getting queued.  But this test is not testing what it says at all so fixing now.  Totally missed this one's check when I was adjusting these.

Reply:
94306da